Vinyl/RequestMatcherRegistry.swift

```diff
@@ -60,75 +59,75 @@ public struct RequestMatcherRegistry {
 // MARK: - Matchers
 
 private struct MethodRequestMatcher: RequestMatcher {
-    func match(aRequest: Request, anotherRequest: Request) -> Bool {
+    func match(request: Request, with anotherRequest: Request) -> Bool {
         // `caseInsensitiveCompare` doesn't support an optional, to prevent unwrapping and perform more than one comparison we can capitalize both methods and rely on optionals to do the hard-work
```

Comment:
normally here, I would prefer the same approach with `Equatable`. `lhs` and `rhs`.